templates/components/_nav_desktop.html.twig

```diff
@@ -0,0 +1,163 @@
+        </li>
+        <li class="text--medium-small">
+            <a href="{{ path('page_campus') }}">Campus</a>
+        </li>
+        <li class="text--medium-small">
+            <a href="{{ path('app_search_events') }}">Evenements</a>
+        </li>
+        {% if is_granted('ROLE_USER') %}
+            <li>
+                <a class="text--medium-small" href="{{ path('homepage') }}">Devenir adhérent(e)</a>
+            </li>
+        {% endif %}
+        {% if app.user and (is_granted('ROLE_ADHERENT') or app.user.isHost or app.user.getMemberships|length) %}
+            <li>
+                <a class="text--medium-small" href="{{ path('homepage') }}">Mon comité</a>
+            </li>
+        {% endif %}
+        {% if is_granted('ROLE_REFERENT') %}
+            <li>
+                <a class="text--medium-small" href="{{ path('app_referent_users') }}">Espace référent</a>
+            </li>
+        {% endif %}
+        {% if is_granted('ROLE_BOARD_MEMBER') %}
+            <li>
+                <a class="text--medium-small" href="{{ path('app_board_member_home') }}">Espace membre du Conseil</a>
+            </li>
+        {% endif %}
+        {% if is_granted('ROLE_COORDINATOR') %}
+            <li class="text--small">
+                <a href="{{ path('app_coordinator_committees', { 's': constant('AppBundle\\Entity\\Committee::PENDING') }) }}">Espace coordinateur régional</a>
+            </li>
+        {% endif %}
+        {% if is_granted('ROLE_PROCURATION_MANAGER') %}
+            <li class="text--small">
+                <a href="{{ path('app_procuration_manager_index') }}">Espace responsable procuration</a>
+            </li>
+        {% endif %}
+        {% if is_granted('ROLE_LEGISLATIVE_CANDIDATE') %}
+            <li class="text--small">
+                <a href="{{ path('app_legislative_candidates_platform') }}" target="_blank">
+                    Espace candidat aux législatives
+                </a>
+            </li>
+        {% endif %}
+        {% if app.user and (is_granted('ROLE_ADHERENT') or app.user.isHost) %}
+            <li>
+                <a class="text--medium-small" href="{{ path('app_documents_index') }}">Mon kit</a>
+            </li>
+        {% endif %}
+        {% if not app.user %}
+            <li class="text--medium-small">
+                <a href="{{ path('donation_index') }}">Donner</a>
+            </li>
+        {% endif %}
+
+        <li {% if not algolia_app_id or not algolia_api_key %}style="display: none;"{% endif %}>
+            <a href="javascript:void(0)" class="je-cherche">
+                {{ include('components/search.html.twig') }}
+            </a>
+        </li>
+    </ul>
+    <div class="l__row">
+        {% if not campaign_expired and not user_is_adherent %}
+            <a href="{{ path('app_membership_register') }}"
+               class="btn b__nudge--right-nano">
+                J'adhère
```

Comment:
`Devenir adhérent(e)`  instead of `J'adhère`. We're moving to infinitive CTAs. No more `I`s.  

Though we're removing this one per the designs, no?